debugger_implementation.tex

```diff
@@ -46,8 +46,9 @@ \section{Checking for Halted Harts}
 \section{Halting} \label{deb:halt}
 
 To halt one or more harts, the debugger selects them, sets \Fhaltreq, and then
-waits for \Fallhalted to indicate the harts are halted before clearing
-\Fhaltreq to 0.
+waits for \Fallhalted to indicate the harts are halted. Then it can clear
+\Fhaltreq to 0, or leave it high to catch a hart that unexpectedly resets while
```

Comment:
though I don't know if we need "unexpectedly". This is how you do it when you do `ndmreset`, and are expecting the hart to reset.

Reply:
Removed "unexpectedly."